Backend/tests/test_user_ctrl.py

```diff
@@ -257,6 +269,60 @@ def test_getRole_fail(self):
     userRole = usr.getRole('test85@mentii.me', dynamodb)
     self.assertIsNone(userRole)
 
+  def test_joinClass(self):
+    print 'Running test_joinClass'
+
+    joinEmail = 'join@mentii.me'
+    password = 'password'
+    activationId = usr.addUserAndSendEmail(joinEmail,password,mail,dynamodb)
+    usr.activate(activationId, dynamodb)
+
+    teacherEmail = 'teacher@mentii.me'
+    activationId = usr.addUserAndSendEmail(teacherEmail,password,mail,dynamodb)
+    usr.activate(activationId, dynamodb)
+
+    classData = {
+      'title' : 'title',
+      'description' : 'desc'
+    }
+    teacherRole = 'teacher'
+    classCtrl.createClass(dynamodb, classData, teacherEmail, teacherRole)
+
+    res = classCtrl.getPublicClassList(dynamodb, joinEmail)
+
+    #confirm response can be created from classSet
+    flaskRes = createResponse(res, 200)
+
+    classes = res.payload['classes']
+    allClassCodes = set()
+    for c in classes:
+      code = c['code']
+      allClassCodes.add(code)
+      joinData = { 'code' : code }
+      res = usr.joinClass(joinData, dynamodb, joinEmail)
+      self.assertFalse(res.hasErrors())
+      self.assertEqual(res.payload['code'], code)
+
+    joined = classCtrl.getClassCodesFromUser(dynamodb, joinEmail)
+    missing = joined - allClassCodes
+    #confirm none missing
+    self.assertFalse(missing)
```

Comment:
So an empty set is the same as false? Not something that you need to change but it just wasn't immediately clear and I wanted to make sure that the test makes sense.